aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -49,39 +49,32 @@ module aptos_framework::account {
 
     const MAX_U64: u128 = 18446744073709551615;
 
-    /// Account already existed
-    const EACCOUNT: u64 = 0;
-    /// Sequence number exceeded the maximum value for a u64
-    const ESEQUENCE_NUMBER_TOO_BIG: u64 = 1;
-    /// The address provided didn't match the `aptos_framework` address.
-    const ENOT_APTOS_FRAMEWORK: u64 = 2;
+    /// Account already existed.
+    const EACCOUNT_ALREADY_EXISTS: u64 = 1;
+    /// Sequence number exceeded the maximum value for a u64.
+    const ESEQUENCE_NUMBER_TOO_BIG: u64 = 2;
     /// The provided authentication had an invalid length
     const EMALFORMED_AUTHENTICATION_KEY: u64 = 3;
-
-    const ECANNOT_CREATE_AT_VM_RESERVED: u64 = 4;
-    const EGAS: u64 = 5;
-    const ECANNOT_CREATE_AT_CORE_CODE: u64 = 6;
-    const EADDR_NOT_MATCH_PREIMAGE: u64 = 7;
-    const EWRITESET_NOT_ALLOWED: u64 = 8;
-    const EMULTI_AGENT_NOT_SUPPORTED: u64 = 9;
-    const EMODULE_NOT_ALLOWED: u64 = 10;
-    const ESCRIPT_NOT_ALLOWED: u64 = 11;
+    /// Cannot create an account because address has been reserved by the system.
+    const ECANNOT_RESERVED_ADDRESS: u64 = 4;
+    /// Transaction's gas limit has been exceeded and thus ran out of gas.
+    const EOUT_OF_GAS: u64 = 5;
+    /// Writesets are not allowed.
+    const EWRITESET_NOT_ALLOWED: u64 = 6;
 
     /// Prologue errors. These are separated out from the other errors in this
     /// module since they are mapped separately to major VM statuses, and are
     /// important to the semantics of the system.
     const PROLOGUE_EINVALID_ACCOUNT_AUTH_KEY: u64 = 1001;
     const PROLOGUE_ESEQUENCE_NUMBER_TOO_OLD: u64 = 1002;
     const PROLOGUE_ESEQUENCE_NUMBER_TOO_NEW: u64 = 1003;
-    const PROLOGUE_EACCOUNT_DNE: u64 = 1004;
+    const PROLOGUE_EACCOUNT_DOES_NOT_EXIST: u64 = 1004;
```

Comment:
Do these take error messages?

Reply:
No these are mapped manually in the errors.rs also changed in this PR.